### ISSUE TITLE
Does not work with ver. 1865

### ISSUE THREAD

**Issue by artooksanen** [marco402/plugin-Rtl433-for-SdrSharp#6]
I copied the files but the plugin does not start. Should it work?

arto

**Comment by marco402** [comment]
Hi artooksanen

I tried sdrsharp1865, no problem for me.
What is your problem? Is it loading or launching the plugin?
You can see the latest radioreference message, spleen tells me that by removing the underline in the folder name rtl_433, I do not have this problem.

[https://forums.radioreference.com/threads/rtl433-plugin-for-sdr-now-available.425703/page-8](url)

Marc

**Comment by artooksanen** [comment]
The problem is that I dont see the plugin anywhere in the UI. I have tried to copy the files in different folders with no change. The installation instructions are a bit unclear (either in? DLL_433? The key?):

> Since version 1830, there is no longer a bin folder, just put the 3 dll either in the plugins folder either in plugins/DLL_433 no need for key. For versions that have sdrsharp.exe in a bin folder.
> In this case, position yourself in C:SDRSharp/plugins
> - Create a folder RTL_433 and place the 3 dlls there.
> - Important:Delete the 3 dlls if they are in bin.
> - Delete the key if it is in plugins.xml.

Does the folder of SDRSharp.exe matter?

arto

**Comment by marco402** [comment]
No, the SDRSharp.exe file does not matter.
The 3 dlls can be directly in the plugins folder or in a subfolder located in plugins.
The important thing is that the path of the plugins corresponds to the path indicated in SDRSharp.config under the key:key="core.pluginsDirectory"
relative path to SDRSharp.

Marc

**Comment by artooksanen** [comment]
Nope, does not work. What is the file Plugins.xml for? There is also file "SDRSharp.exe.Config".

arto

**Comment by marco402** [comment]
The 2 files Plugins.xml and SDRSharp.exe.Config are rests of the older versions SDRSharp, the plugin rtl_433 still loads if I delete them.
I tried on a computer without visual studio, just unzip the zip sdrsharp and add the 3 dlls in plugins with or without the folder RTL_433.
It will then be necessary to launch install-rtlsdr.bat if it is to use an sdr key. 
For information, I am on Windows 10 64bits and I have the .NET runtime 5.0.7 and 5.0.17.
To check you open an admin console and type dotnet --info.
if you do not have version 5, you can install it, the dotnet command gives you the link.

Marc
